### PR TITLE
Enable 360 icon on idle

### PR DESCRIPTION
**Pull request by karimJWP**
### This PR will...
create a carboard available flag 
### Why is this Pull Request needed?
to display 360 icon on idle
### Are there any points in the code the reviewer needs to double check?
is this overkill ?
### Are there any Pull Requests open in other repos which need to be merged with this?
https://github.com/jwplayer/jwplayer-vr-plugin/pull/45
#### Addresses Issue(s):
JW8-643

### CODE REVIEW

File: src/css/controls/flags/cardboard-available.less [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.jwplayer.jw-flag-cardboard-available {
+    .jw-icon-cardboard {
+        display: flex;
+    }
+
+    &.jw-state-idle {
```
Comment (egreaves) [inline src/css/controls/flags/cardboard-available.less]: this duplicates the rule in `cast-available.less`. We should remove it from both and put it in `states.less`.

**Comment by johnBartos** [comment]
@karimJWP Can you screenshot or provide a config where we can see what this looks like?

**Comment by karimJWP** [comment]
@johnBartos on android chrome http://0.0.0.0:3000/jwplayer-commercial/test/squash/bootstrap.html?primary=html5&edition=ads&feature=vr_360/360_playlists&suite=VR_360

**Comment by karimJWP** [comment]
@johnBartos unnamed

**Pull request merged**